demo/test/bitstyles_phoenix_demo_web/bitstyles_phoenix_test.exs

```diff
@@ -12,4 +12,19 @@ defmodule BitstylesPhoenixWebDemo.BitstylesPhoenixTest do
     |> find(Query.css(".e2e-form-de", count: 1))
     |> assert_has(Query.css(".u-fg--warning", text: "muss ausgefüllt werden"))
   end
+
+  feature "allows icon config and can still render inline", %{session: session} do
+    session
+    |> visit("/")
+    |> find(Query.css(".e2e-external-svg-icon use", count: 1), fn element ->
+      assert {width, height} = Element.size(element)
+      assert width > 0
+      assert height > 0
+    end)
+    |> find(Query.css(".e2e-inline-svg-icon use", count: 1), fn element ->
+      assert {width, height} = Element.size(element)
+      assert width > 0
+      assert height > 0
+    end)
+  end
```

Comment:
👍  Nice, didn't realize you intended to use the demo app for tests until now. Nice approach!

One suggestion though: Instead of making the demo app have its own test suite, you could integrate these tests into the main test suite? i.e. something like [this]. At least I don't see a reason why it wouldn't work like this?

Reply:
checked the circleci config: I guess you put some thoughts into this, so I'm curious: What's the benefit of having two jobs? I realize that the demo tests are browser-based, but in the end for any development on bitstyles_phoenix you'd want contributors to run both test suites anyway, right? IMO would be a bit more convenient if I wouldn't have to run two test suites.

same for the formatter btw, you could just add `demo` to `.formatter.exs`

Reply:
hmm 🤔 technically possible, but then I have to find a way on how to configure wallaby without a project next to it ...
There is all this niceties (like automatically starting a server, compiling assets etc) that I get with this setup. 
In the end if you do not want to call two things I could create a `script` or something similar...

Reply:
🤔 

> configure wallaby without a project next

I guess what I'm suggesting is to simply make the `demo` project part of bitstyles_phoenix, move its configuration into toplevel `config` files and only import it for `dev`/`test`?

But this may be too much work, it's also great as it is. Top-level `script` folder: Sure, why not

Reply:
https://github.com/bitcrowd/chromic_pdf/blob/main/config/config.exs#L14-L16

Reply:
It should also serve as demo config project, so I want to keep it not super far from a generated phoenix project...